src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -1149,6 +1161,16 @@ public function single_row( $item ) {
 
 					echo '</div>';
 
+					/**
+					 * Fires after plugin row meta.
+					 *
+					 * @since 6.4.0
+					 *
+					 * @param string $plugin_file Refer to {@see 'plugin_row_meta'} filter.
+					 * @param array  $plguin_data Refer to {@see 'plugin_row_meta'} filter.
```

Comment:
```suggestion
					 * @param array  $plugin_data Refer to {@see 'plugin_row_meta'} filter.
```
Bah, I missed the typo! :joy: